tools/taskcluster/schedule-master-build.py

```diff
@@ -33,15 +33,17 @@ def generate_unit_test_task(buildTaskId):
 	return taskcluster.slugId(), generate_task(
 		name = "(Focus for Android) Unit tests",
 		description = "Run unit tests for Focus/Klar for Android.",
-		command = 'echo "--" > .adjust_token && ./gradlew --no-daemon clean test',
+		command = 'echo "--" > .adjust_token && ./gradlew '
+				  '--no-daemon clean test',
 		dependencies = [ buildTaskId ])
 
 
 def generate_code_quality_task(buildTaskId):
 	return taskcluster.slugId(), generate_task(
 		name = "(Focus for Android) Code quality",
 		description = "Run code quality tools on Focus/Klar for Android code base.",
-		command = 'echo "--" > .adjust_token && ./gradlew --no-daemon clean detektCheck ktlint lint pmd checkstyle spotbugs',
+		command = 'echo "--" > .adjust_token && ./gradlew '
+				  '--no-daemon clean detektCheck ktlint lint pmd checkstyle spotbugs',
 		dependencies = [ buildTaskId ])
 
```

Comment:
nit: The changes in this file are only line breaks now and are not really needed. :)